catalog/resource_grants.go

```diff
@@ -210,7 +143,7 @@ var mapping = securableMapping{
 		"APPLY_TAG":      true,
 		"BROWSE":         true,
 	},
-	"view": {
+	"view": { // TODO: Is this supported as it is not documented ?
```

Comment:
you are correct - this is done via `table` securable

Reply:
Ok, so, should I keep the `view` for backwards compat and just pass it to `table` similarly to `model` --> `function`, or remove these args ?.

Ditto `materialized_view` above.

Reply:
Follow up question: The newly added `databricks_grant` does not support these as I followed the api docs, so if you choose to keep `view` as an arg here should I add it to `databricks_grant` also for consistency ?

Reply:
`view` and `materialized_view` have never worked (the API will error out), so should be safe to remove - you just need to add these permissions back to `table` mapping

Reply:
Removed